src/styles/product.scss

```diff
@@ -0,0 +1,72 @@
+.product {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 30px 14px 14px;
+  margin-bottom: 50px;
+  width: $card-width;
+  height: $card-height;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  &__image {
+    height: 134px;
+    width: 160px;
+    margin-bottom: 45px;
+  }
+  &__name {
+    @include nameFont;
+  }
+  &__code {
+    align-self: start;
+    @include smallFont;
+    margin-top: 5px;
+  }
+  &__rate {
+    display: flex;
+    justify-content: space-between;
+    width: 100%;
+    margin-top: 15px;
+
+    &-reviews {
+      @include smallFont;
+      width: 100%;
+      text-align: end;
+    }
+  }
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: baseline;
+    width: 100%;
+    margin-top: 20px;
+    &-title {
+      @include smallFont;
+    }
+    &-amount {
+      @include priceFont;
+    }
+  }
+  &__button {
+    margin-top: 20px;
+    height: 40px;
+    width: 100%;
```

Comment:
don't use width 100% automatically everywhere, if it doesn't affect the appearance, remove it

Reply:
remove width where you don't use it